src/components/Technologies/CategorySelector/TypeSelector.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import styles from "./CategorySelector.module.scss";
+import { Select, ConfigProvider, ThemeConfig, theme } from "antd";
+import { packTypeNames } from "../PackConstants";
+import { useColorMode } from "@docusaurus/theme-common";
+
+interface TypeSelectorProps {
+  categories: string[];
+  selected: string[];
+  selectCategory: (category: string[]) => void;
+}
+
+export default function TypeSelector({ categories, selected = [""], selectCategory }: TypeSelectorProps) {
```

Comment:
selectCategory should be called `onChange`
`selected` should be `value`
`categories` should be options. 
Please standardize these under a single component.
There are too many specific components where it can be a generic one.